docs/nodes/credentials/Spotify/README.md

```diff
@@ -0,0 +1,19 @@
+# Spotify
+You can find information about the operations supported by Spotify node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.spotify) page. You can also browse the source code of the node on [Github](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Spotify).
+
+# Authentication
+
+## Pre-requisites
+
+Create a [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
+
+## Steps
+
+1. Open your Spotify for Developers dashboard.
+2. Create an app.
+3. On the app's dashboard, copy the Client ID and the Client Secret.
+4. Open the app settings and add the Callback URL provided by n8n to the Redirect URIs list.
+5. Use the Client ID and the Client Secret to authenticate with n8n.
+
+
+![The Spotify App Dashboard](https://i.imgur.com/qmjR3xq.gif)
```

Comment:
Could you please make sure that this image exists locally and loads from there (as opposed to from imgur)?